src/shared/components/plots/ScatterPlotUtils.ts

```diff
@@ -9,3 +9,45 @@ export function getRegressionComputations(
 } {
     return regression.linear(data);
 }
+
+export const makeMultilineAxisLabel = (
+    axisLabel: string | undefined,
+    charLimit: number
+): string => {
+    return !axisLabel
+        ? ''
+        : axisLabel.length > charLimit
+        ? generateLineChunks(axisLabel, charLimit).join('\n')
+        : axisLabel;
+};
+
+export const getNumberOfNewlines = (text: string): number => {
+    const regex = /\n/g;
+    const matches = text.match(regex);
+    return matches ? matches.length : 0;
+};
+
+const generateLineChunks = (axisLabel: string, charLimit: number): string[] => {
+    const wordsFollowedBySpaces = axisLabel.match(/([^\s]+\s)/g);
+
+    const result = wordsFollowedBySpaces
```

Comment:
Rather than using a ternary here, just put an early return for if wordsFollowedBySpaces is falsey

```suggestion
    if (!wordsFollowedBySpaces) {
        return [];
    }
    const result = wordsFollowedBySpaces
```